tests/Microsoft.Bot.Builder.Tests/ShowTyping_MiddlewareTests.cs

```diff
@@ -109,5 +154,38 @@ private void ValidateTypingActivity(IActivity obj)
                 throw new Exception("Activity was not of type TypingActivity");
             }
         }
+
+        private class MockBotIdentityMiddleware : IMiddleware
```

Comment:
This works, but I think it may be cleaner to make a helper SetTurnContextClaim method instead of a middleware and call that method from within the TestFlow callback, would that work?